geomesa-utils/src/main/resources/org/locationtech/geomesa/geomesa-site.xml.template

```diff
@@ -96,6 +96,31 @@
     <!-- /geomesa.batchwriter -->
 
     <property>
+        <name>geomesa.mapreduce.splits.max</name>
+        <value></value>
+        <description>Set the absolute number of splits instead of basing it on the number of tablet servers.
+            Setting this value overrides geomesa.mapreduce.splits.tserver.max.
+
+            Making this value final without setting a value greater than zero may cause unexpected issues.
+        </description>
+        <final>false</final>
+    </property>
+
+    <property>
+        <name>geomesa.mapreduce.splits.tserver.max</name>
+        <value></value>
+        <description>Set the max number of splits per tablet server when configuring a mapper. By default this value is
+            calculated using Accumulo's AbstractInputFormat.getSplits method which creates a split for each range. In
+            some scenarios this may create an undesirably large number of splits.
+
+            This value is overwritten by geomesa.mapreduce.splits.max if it is set.
+
+            Making this value final without setting a value greater than zero may cause unexpected issues.
```

Comment:
yeah, these make sense to me - can you also add them to the documentation here:
http://www.geomesa.org/documentation/user/accumulo/configuration.html